libsolidity/interface/StandardCompiler.cpp

```diff
@@ -263,7 +263,7 @@ bool isBinaryRequested(Json::Value const& _outputSelection)
 	// This does not include "evm.methodIdentifiers" on purpose!
 	static vector<string> const outputsThatRequireBinaries = vector<string>{
 		"*",
-		"ir", "irOptimized",
+		"ir", "irAstJson", "irOptimized", "irOptimizedAstJson",
```

Comment:
We should document these new outputs under the [Standard JSON output description](https://docs.soliditylang.org/en/v0.8.20/using-the-compiler.html#output-description).

Well, in the interest of getting this PR merged quickly, I'd say it would be fine to do it separately. But then, please at least drop a comment in #9590 mentioning that this part is not done.

Reply:
I added some documentation, let me know if it's in line with what you had in mind!